R/simulate_design.R

```diff
@@ -112,6 +112,8 @@ simulate_design <- function(..., sims = 500) {
     parameters_df <- rbind_disjoint(parameters_df_list)
     parameters_df <- data.frame(lapply(parameters_df, type_convert), stringsAsFactors = FALSE)
     
+    simulations_df <- simulations_df[ , c(colnames(parameters_df), setdiff(colnames(simulations_df), colnames(parameters_df))), drop = FALSE]
```

Comment:
there are enough of these scattered through to  justify a helper function - eg 

ddreorder = function(a,b, n1=colnames(a), n2=colnames(b)) c(n1, setdiff(n2, n1))

Reply:
will do